app/migrations/seed_dev.py

```diff
@@ -1,161 +1,371 @@
 
-def get_or_create_token(address, name, symbol):
-    instance = Token.query.filter_by(address=address).first()
-    print(instance)
-    if instance:
-        return instance
-    else:
-        instance = Token(address=address,
-                         name=name,
-                         symbol=symbol)
+from server.utils.user import create_transfer_account_user
 
-        db.session.add(instance)
-        return instance
+def load_account(address, amount_wei):
+    from web3 import (
+        Web3,
+        HTTPProvider
+    )
 
+    w3 = Web3(HTTPProvider(config.ETH_HTTP_PROVIDER))
+
+    tx_hash = w3.eth.sendTransaction(
+        {'to': address, 'from': w3.eth.accounts[0], 'value': amount_wei})
+    return w3.eth.waitForTransactionReceipt(tx_hash)
+
+
+def get_or_create_reserve_token(deploying_address, name, symbol):
+
+    reserve_token = Token.query.filter_by(symbol=symbol).first()
+    if reserve_token:
+        return reserve_token
 
-def get_or_create_organisation(name, token):
-    instance = Organisation.query.filter_by(name=name).first()
-    print(instance)
-    if instance:
-        return instance
     else:
-        instance = Organisation(name=name, token=token)
+        reserve_token_address = bt.deploy_and_fund_reserve_token(
+            deploying_address=deploying_address,
+            name=name,
+            symbol=symbol,
+            fund_amount_wei=0
+        )
 
-        db.session.add(instance)   
-        return instance
+        reserve_token = Token(address=reserve_token_address, name=name, symbol=symbol)
+        reserve_token.decimals = 18
+
+        db.session.add(reserve_token)
+
+        return reserve_token
+
+
+def create_or_get_reserve_token(
+        deploying_address,
+        reserve_token,
+        name, symbol, reserve_ratio_ppm,
+        reserve_deposit_wei,
+        issue_amount_wei,
+        registry_address=None,
+        exchange_contract=None):
+
+    if registry_address is None:
+        registry_address = bt.deploy_exchange_network(deploying_address)
+
+    smart_token_result = bt.deploy_smart_token(
+        deploying_address=deploying_address,
+        name=name, symbol=symbol, decimals=18,
+        reserve_deposit_wei=reserve_deposit_wei,
+        issue_amount_wei=issue_amount_wei,
+        contract_registry_address=registry_address,
+        reserve_token_address=reserve_token.address,
+        reserve_ratio_ppm=reserve_ratio_ppm
+    )
+
+
+    smart_token_address = smart_token_result['smart_token_address']
+    subexchange_address = smart_token_result['subexchange_address']
+
+    smart_token = Token(address=smart_token_address, name=name, symbol=symbol)
+    smart_token.decimals = 18
+
+    db.session.add(smart_token)
+
+    if exchange_contract is None:
 
+        exchange_contract = ExchangeContract(
+            blockchain_address=subexchange_address,
+            contract_registry_blockchain_address=registry_address
+        )
 
-def get_or_create_admin_user(email):
```

Comment:
this diff looks real bad because i re-ordered things a bit and it's apparently confused!